test/lib/fetch.test.ts

```diff
@@ -23,4 +24,8 @@ describe('fetch', () => {
       superagent,
     }, '/homePageContent', '');
   });
+  it('successfully runs fetchPost', async () => {
+    r = await fetch.fetchPost(superagent, { token: '' }, { title: '', comments: '', type: '' });
+    expect(r).toBe('');
```

Comment:
expect(typeof status).toBe('number')

Reply:
It's still giving the same error saying "TypeError: superagent.post is not a function

 it('successfully runs fetchPost', async () => {
    const { status } = await fetch.fetchPost(superagent, { token: '' }, { title: '', comments: '', type: '' });
    expect(typeof status).toBe('number');
  });